app/config/parameters.yml.dist

```diff
@@ -17,7 +17,7 @@ parameters:
     env(DATABASE_USER): root
     env(DATABASE_PASSWORD):
     env(DATABASE_SERVER_VERSION): mariadb-10.2.12
-    siso_search.solr.host: localhost
+    siso_search.solr.host: solr
```

Comment:
It should be possible to inject these settings via environment variables ( so use env() function as above and define `siso_search.solr.[host|port|core|path] in https://github.com/ezsystems/ezcommerce-advanced/blob/master/app/config/default_parameters.yml

localhost should still be default value.

Reply:
Maybe this change could cause issues for manual installation in case a user just use the default values (pressing return)? 
We wil at least add a hint in the documentation.

Reply:
With @jheba latest change, 'localhost' is again default value and should introduce no changes/issues in regards to manual installation, right @fdege ?

This conversion can be marked as resolved, right ?

Reply:
Yes, we can close it